.github/workflows/update-licenses.yml

```diff
@@ -12,22 +16,31 @@ jobs:
   run:
     name: Recompute licenses & update PR
     runs-on: ubuntu-latest
+    env:
+      BRANCH: ${{ inputs.branch || github.ref_name  }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ env.BRANCH }}
 
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
           go-version-file: "${{ github.workspace }}/go.mod"
           cache: false
 
-      - name: recompute licenses
+      - name: Recompute licenses
         run: make recompute-licenses
 
-      - uses: stefanzweifel/git-auto-commit-action@v4.1.1
-        with:
-          commit_message: |
-            Updated dependency licenses
-
-            dependabot skip
-          branch: ${{ github.head_ref }}
+      - name: Commit as needed
+        run: |
+          if [[ $(git diff --stat) != '' ]]; then
+            echo 'Committing changes'
+            git config user.email "akobot@ako-team-fake.mongodb.com"
```

Comment:
Is this parameter mandatory?

Reply:
Yes, it seems git complains otherwise. Also it is best we know it was the bot that did the PR.